src/applications/static-pages/homepage/HomepageSearch.jsx

```diff
@@ -73,6 +73,24 @@ const HomepageSearch = () => {
       e.target.value,
     )}&t=${false}`;
 
+    // Record the analytic event.
+    recordEvent({
+      event: 'view_search_results',
+      action: 'Homepage - Search',
+      'search-page-path': searchUrl,
+      'search-query': e.target.value,
+      'search-results-total-count': null,
+      'search-results-total-pages': null,
+      'search-selection': 'All VA.gov',
+      'search-typeahead-enabled': false,
+      'search-location': 'Homepage Search',
+      'sitewide-search-app-used': false,
+      'type-ahead-option-keyword-selected': null,
+      'type-ahead-option-position': null,
+      'type-ahead-options-list': null,
+      'type-ahead-options-count': null,
+    });
+
```

Comment:
- [ ] "Popular on VA.gov" AC fell behind naming. This section is now called 'Top Pages' and 'Popular on VA.gov' should get renamed in the action.

Reply:
I've pushed an update which should bring the GA event in line with the new name for the section.

Reply:
Screenshot 
<img width="811" alt="Screenshot 2022-11-30 at 2 53 51 PM" src="https://user-images.githubusercontent.com/732460/204905906-b5a0771e-350e-43e9-8371-d87157e5e5db.png">

Reply:
For posterity, just noting that the change mentioned in this comment thread is implemented in the [related content-build PR](https://github.com/department-of-veterans-affairs/content-build/pull/1380).